.travis.yml

```diff
@@ -55,16 +56,16 @@ matrix:
       addons:
           apt:
               sources: *all_sources
-              packages: ['clang-5.0', 'libstdc++-7-dev']
-      env: MATRIX_EVAL="CC=clang-5.0 && CXX=clang++-5.0 && COVERAGE=Off"
+              packages: ['clang-6.0', 'libstdc++-7-dev']
+      env: MATRIX_EVAL="CC=clang-6.0 && CXX=clang++-6.0 && COVERAGE=Off"
     - os: linux
       dist: xenial
       compiler: clang
       addons:
           apt:
               sources: *all_sources
-              packages: ['clang-6.0', 'libstdc++-7-dev']
-      env: MATRIX_EVAL="CC=clang-6.0 && CXX=clang++-6.0 && COVERAGE=Off"
+              packages: ['libstdc++-7-dev']
```

Comment:
I think it should be: `packages: ['clang-8.0', 'libstdc++-8-dev']`

Reply:
It's actually `clang-8` (I know, weird), and already fixed.